modules/websocket/lws_client.cpp

```diff
@@ -34,7 +34,7 @@
 #include "core/io/stream_peer_ssl.h"
 #include "tls/mbedtls/wrapper/include/openssl/ssl.h"
 
-#if defined(MINGW_ENABLED) || defined(_MSC_VER)
+#if !defined(MINGW_ENABLED) && defined(_MSC_VER)
```

Comment:
No need to check `!defined(MINGW_ENABLED)`, the two are mutually exclusive. `_MSC_VER` means that you're using MSVC, which is not MinGW.